### DISCUSSION TITLE
No series count for fetch book data on upload

### DISCUSSION THREAD

**Discussion by lukeIam** [advplyr/audiobookshelf#2558] [Q&A] [answered]
Is this a bug or was it never part of https://github.com/advplyr/audiobookshelf/pull/2333 ?

![image](https://github.com/advplyr/audiobookshelf/assets/5115160/ab7d51a2-c98d-4143-9a27-11c78c014962)
![image](https://github.com/advplyr/audiobookshelf/assets/5115160/de5a5ed4-ff52-4b22-8580-794e985a5e4a)

**Comment by advplyr** [comment, marked as answer]
It's not part of it. This is just for creating the folder names. It may be better to start the title with the series number but it may be confusing.

**Comment by lukeIam** [comment]
maybe this can be achieved together with https://github.com/advplyr/audiobookshelf/pull/1521 - dreaming of somebody uploads an audiobook and it gets (semi-)automatically tagged and placed in the right folders (including series#)
